middleware/middleware-1.3.16.md

```diff
@@ -0,0 +1,72 @@
+
+## Compatibility improvement: Fiskal Cloud Connector on Linux
+During the last two weeks, our partners informed us that the FCC could not be successfully installed on some Linux distributions without manually making some files executable. We have changed this behavior so that no manual action is needed anymore when setting up a _Swissbit Cloud_ or a _Deutsche Fiskal_ TSE on Linux.
+
+## Bug fix: Deleting logs on fiskaly TSEs
+To ensure equal functionality in all our TSE implementations, we've updated our fiskaly SCU to now mark logs that have been exported during a daily closing receipt as deleted. This resolves two issues:
+- Previously, the fiskaly SCU always did a full export when daily-closing receipts were processed, and stored the results in the database. When a range-based `/Journal` export of those TAR files was called, the response could hence include more logs than requested. 
+- Due to the full export, the duration of daily-closing receipts on fiskaly SCUs increased with each processed transaction, ultimately making the process very slow when many transactions were performed. 
+
+Summarized, the fiskaly SCU now behaves as our other TSE implementations and only exports TAR logs that were generated since the last daily-closing receipt. However, there's one exception: in case there are open transactions, it's not possible to mark any transactions as deleted; the open transactions need to be closed first.
+
+## Stability improvement: Add flag to ignore transaction mismatches at daily-closing receipts
+In rare cases (e.g. due to power or network failures), TSE transactions could be marked as finished on the TSE, but the Middleware still kept them open in our internal database. As these "open" transactions must be closed before a daily-closing receipt can be executed, this effectively blocked the Middleware from performing this special receipt.
+
+To resolve this problem, we've introduced the _ftReceiptCaseFlag_ `0x0000000020000000` (as described [here](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftreceiptcase#ftreceiptcaseflag)). When this flag is sent with the daily-closing receipt, the Middleware will check if transactions that are marked open in our database are actually open in the TSE; if this is the case, a journal entry is made and the database is updated.
+
+**Please note that this flag should be be used per default, as it may hide power configuration issues or network problems that lead to incomplete DSFinV-K exports**. However, it may be useful in these cases to unblock users of cash registers in case this scenario occurred.
```

Comment:
```suggestion
**Please note that this flag shouldn't be used per default, as it may hide power configuration issues or network problems that lead to incomplete DSFinV-K exports**. However, it may be useful in these cases to unblock users of cash registers in case this scenario occurred.
```